specs/am/architecture-management-spec.html

```diff
@@ -800,7 +812,7 @@ <h2>Samples</h2>
       <p>
         See
         <a
-          href="http://open-services.net/wiki/architecture-management/OSLC-Architecture-Management-2.0-Appendix-A%3A-Samples"
+          href="https://archive.open-services.net/wiki/architecture-management/OSLC-Architecture-Management-2.0-Appendix-A_-Samples/index.html"
```

Comment:
Maybe let's leave the original URL? Otherwise be very hard to change redirects later.

Reply:
I'll use https://open-services.net/wiki/architecture-management/OSLC-Architecture-Management-2.0-Appendix-A_-Samples/